src/module_wrap.cc

```diff
@@ -623,23 +639,11 @@ Maybe<URL> ResolveModule(Environment* env,
   return Nothing<URL>();
 }
 
-Maybe<URL> ResolveDirectory(Environment* env,
-                            const URL& search,
-                            PackageMainCheck check_pjson_main) {
-  if (check_pjson_main) {
-    Maybe<URL> main = ResolveMain(env, search);
```

Comment:
It seems this is never called now.

This is likely my fault - rushed the refactoring a bit last time. Will take another look.